install_config/aggregate_logging.adoc

```diff
@@ -654,6 +654,50 @@ $ oc label nodes --all logging-infra-fluentd=true
 
 endif::openshift-origin[]
 
+When you make changes to any part of the EFK stack, specifically Elasticsearch
```

Comment:
Why is this section under the title _Fluentd_? It covers more than just that tool.